### ISSUE TITLE
a11y: Table field navigation

### ISSUE THREAD

**Issue by pearlyong** [opengovsg/FormSG#5504]
- On iOS Voiceover, when in the first column, it reads out the entire question again
- on MacOS Voiceover, when entering the table field, focus jumps to the second column
- on NVDA, when attempting to open the combobox, focus jumps to bottom of page
- Can we change Comboboxes to normal dropdown fields to avoid problems mentioned in #5502 and above

**Comment by staceytan1998** [comment]
**Issue**

When using the arrow keys on a screen reader to navigate, the user is unable to ""click"" on the drop-down menu using the Space and Enter key.

The drop-down menu is only accessible when navigating using the Tab key to navigate.

**Suggestion**

This issue could be caused by an incomplete tag, or some other issue causing the drop-down menu element to not reliably be detectable by screen readers.

**Screenshot**

https://drive.google.com/file/d/1m-ldDfDuLKE1DH5QrcABAL_8E9O_JQD4/view?usp=share_link

**Comment by justynoh** [comment]
Not quite sure if the table issue is our fault... trying it out even on the [accessible table](https://www.w3.org/WAI/ARIA/apg/patterns/table/examples/table/) provided by the w3 docs, voiceover on mac has the same behavior. I wonder if it's a bug in voiceover rather than us, at least in the case of voiceover on macos. Can we test to see if the behavior of iOS voiceover is the same on formsg as it is in the w3 docs?

https://user-images.githubusercontent.com/25571626/218725330-42fbc01c-ccd1-4b65-9093-ae704634d216.mov

For NVDA, the issue will likely be resolved once we resolve the issue with dropdowns.

**Comment by staceytan1998** [comment]
The behaviour of the table under [this heading](https://webaim.org/techniques/forms/advanced#:~:text=by%20this%20element.%22-,Handling%20Multiple%20Labels,-Because%20the%20id) seems ideal! 



https://user-images.githubusercontent.com/99157017/228179286-a997ccf4-657a-4e1e-98ae-f03226513ca7.mov
